kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -488,19 +510,23 @@ define([
             this.dataTable.DataTable().row.add(jobState).draw();
         }
 
+        eventToRow(e) {
+            return $(e.target).closest('tr');
+        }
+
         /**
          * Click handler for the table. Rows can be expanded to show job details
          * and clicked again to hide them.
          *
-         * @param {Event} e - row click event
+         * @param {DOM element} $currentRow - jQuery object for the TR DOM element
          * @returns {Promise} that resolves when the appropriate row action completes
          */
-        showHideChildRow(e) {
-            const $currentRow = $(e.target).closest('tr');
-            const $table = $(e.target).closest('table');
+        showHideChildRow($currentRow) {
```

Comment:
Change the `showHideChildRow` function to take a jQuery row object as input, so it can be used without needing an event to trigger it.